src/renderer/App.css

```diff
@@ -40,6 +40,8 @@
   overflow-y: scroll;
   block-size: 40vh;
   display: block;
+  padding-inline: 16px;
+  margin-block: 16px;
 }
```

Comment:
Prevents a horizontal scroll bar appearing on mobile screen sizes.
```suggestion
  overflow-wrap: break-word;
}
```

Reply:
Thanks for the feedback. This is how it looks now.

![Screenshot_20240514_174216](https://github.com/FreeTubeApp/FreeTube/assets/84899178/bc1dec48-0c7a-4390-a8c9-f28e49147bee)